app/models/edit_course_options.rb

```diff
@@ -0,0 +1,27 @@
+class EditCourseOptions
+  def initialize(course)
+    @course = course
+  end
+
+  def entry_requirements
+    Course::ENTRY_REQUIREMENT_OPTIONS
+      .reject { |a| a.include? %i[not_set not_required] }
```

Comment:
This doesn't look quite right to me, and I'm not even sure if it works. Maybe you want:

```suggestion
      .reject { |k,v| k.in? %[not_set not_required] }
```